src/app/components/user/users.jsx

```diff
@@ -163,6 +183,27 @@ function ChplUsers({
     return null;
   };
 
+  const displayUserEdit = (userToEdit) => {
+    if (userToEdit.cognitoId) {
+      return (
+        <ChplCognitoUserEdit
+          user={userToEdit}
+          errors={errors}
+          dispatch={handleDispatch}
+        />
+      );
+    } if (userToEdit.userId) {
```

Comment:
```suggestion
    } 
    if (userToEdit.userId) {
```

Either a newline, or an `else if`. Putting the if on the same line as the end of the previous block is odd